deploy/template.yaml

```diff
@@ -3356,10 +3359,23 @@ Resources:
     Type: AWS::Serverless::StateMachine
     Properties:
       Name: !Sub "F2F-YotiCallback-stepfunction-${AWS::StackName}"
+      AutoPublishAlias: live
+      AutoPublishAlias: live
+      DeploymentPreference:
+        Type: !Ref StepFunctionsDeploymentPreference
+        Interval: 10
+        Percentage: 10
+        Alarms:
+          - !Ref YotiCallbackStateMachineFailedCanary
```

Comment:
Is it worth adding a conditional like for lambda canaries here too so ? e.g.
        Alarms: !If
          - UseCanaryDeploymentAlarms
          - - !Ref ALARM1
            - !Ref ALARM2
          - !Ref AWS::NoValue
          
Conditions:  
  UseCanaryDeploymentAlarms: !Or
      - !Not [ !Equals [ !Ref StepFunctionsDeploymentPreference, ALL_AT_ONCE ]]
      - !Not [ !Equals [ !Ref LambdaDeploymentPreference, AllAtOnce ]]

Reply:
DONE.